File: Atcoder_#359/javascript/b_couples.js
function Main(input) {
  const lines = input.trim().split('\n');
  const n = parseInt(lines[0])
  const a = lines[1].split(" ").map(Number);

  const num_of_people = 2 * n;
  let ans = 0

  for (let i = 0; i < num_of_people - 2; i++) {
    if (a[i] === a[i + 2]) {
      ans += 1;
    }
  }

  console.log(ans);
}

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin', 'utf8');
Main(input);

File: Atcoder_#359/ruby/b_couples.rb
n = gets.to_i
a = gets.split.map(&:to_i)

num_of_people = 2 * n;
ans = 0
i = 0

(num_of_people - 2).times do
  if a[i] == a[i + 2]
    ans += 1
  end
  i += 1
end

puts ans
